packages/amplify-cli/src/plugin-helpers/platform-health-check.ts

```diff
@@ -0,0 +1,190 @@
+import path from 'path';
+import chalk from 'chalk';
+import { PluginInfo } from '../domain/plugin-info';
+import { PluginPlatform } from '../domain/plugin-platform';
+import { readJsonFileSync } from '../utils/readJsonFile';
+
+type PluginDescription = {
+  name: string;
+  type: string;
+  packageName: string;
+  packageVersion?: string;
+};
+
+const officialPlugins: { [key: string]: PluginDescription } = {
```

Comment:
Can we move it from code the the CLI's ```package.json```, since the other half of the information is coming from there as well. We could have an ```amplify``` section with ```officialPlugins``` member and this information. The maintenance cost would be lower since when we add a new package to the CLI it is obvious that ```officialPlugins``` needs an update as well.

Reply:
Can we automate the generation of official plugin for every build. I don't like the prospect of maintaining one more list

Reply:
@attilah that sounds like a good idea. I will give it a try. 
@yuth I'm not sure about auto generation, a little concerned about having too many moving parts in the plugin platform.

Reply:
move that it to the package.json in the latest update.